json/Arles_Text.txt

```diff
@@ -6036,423 +6036,423 @@
 		"jp_text": "ダメです……このままじゃ……！",
-		"tr_text": "",
+		"tr_text": "Is this... the end...?",
 		"fileID": 2
 	},
 	{
 		"eventNo": "23",
 		"jp_name": "アーレスカイゼル",
 		"tr_name": "Ares Kaiser",
 		"jp_text": "させません！！",
-		"tr_text": "",
+		"tr_text": "I will not let you!!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "24",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "うぅ……やっと止んだか……<%br>みんな、無事か？",
-		"tr_text": "",
+		"tr_text": "Ugh... It finally stopped...<%br>Is everyone alright?",
 		"fileID": 2
 	},
 	{
 		"eventNo": "25",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "あ、あれ……？　ぜんぜん平気だぞ……？",
-		"tr_text": "",
+		"tr_text": "What the heck...? I'm OK...?",
 		"fileID": 2
 	},
 	{
 		"eventNo": "26",
 		"jp_name": "アーレスカイゼル",
 		"tr_name": "Ares Kaiser",
 		"jp_text": "無事、のようです……ね……<%br>よ、よかった……",
-		"tr_text": "",
+		"tr_text": "It seems... you all are... safe...<%br>I... am glad...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "27",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "カイゼルさん！？<%br>どうしたんですか！？　カイゼルさん！",
-		"tr_text": "",
+		"tr_text": "Kaiser?!<%br>What happened?! Kaiser!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "28",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "ひどいケガ……<%br>まさか、私たちをかばったの……？",
-		"tr_text": "",
+		"tr_text": "She's badly injured...<%br>You put yourself in harm's way to protect us...?",
 		"fileID": 2
 	},
 	{
 		"eventNo": "29",
 		"jp_name": "アーレスカイゼル",
 		"tr_name": "Ares Kaiser",
 		"jp_text": "……これでいいのです。<%br>わたくしの希望は……守られたのですから……",
-		"tr_text": "",
+		"tr_text": "...This is fine.<%br>My wishes... have been preserved...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "30",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "お……おい……カイゼル……？<%br>しっかりしろよ！",
-		"tr_text": "",
+		"tr_text": "Huh... Hey... Kaiser...?<%br>Stay with us, y'hear!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "31",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "……まだ息はあるわ。すぐにレスタを……",
-		"tr_text": "",
+		"tr_text": "...She's still breathing.<%br>She needs immediate Resta treatment...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "32",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "待て、アネット。機甲種が集まってきてる。<%br>カイゼルの手当ては安全な場所へ移動してからだ。",
-		"tr_text": "",
+		"tr_text": "Hold up, Annette. There's Mechs outside. We can<%br>take care of Kaiser after we get her to safety.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "33",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "リーダー、まずは機甲種を追い払うぞ！",
-		"tr_text": "",
+		"tr_text": "Leader, let's take out those Mechs!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "？？？",
 		"tr_name": "???",
 		"jp_text": "……なかなかやるじゃないか。",
-		"tr_text": "",
+		"tr_text": "...This is child's play.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "？？？",
 		"tr_name": "???",
 		"jp_text": "まさか俺の全力射撃を耐えきるなんてね。<%br>だが、今度はどうかな？",
-		"tr_text": "",
+		"tr_text": "They cannot withstand my full-power bombardment.<%br>Still, what about next time?",
```

Comment:
Really wasn't sure about the second sentence here. Maybe the four scenes they haven't put in yet would add context, I have no idea what 'next time' he might be referring to.

Reply:
`Though, perhaps another time?`
(we cross paths again) type of ordeal

Reply:
I think I can work with that